### DISCUSSION TITLE
does not recognize component path

### DISCUSSION THREAD

**Discussion by picli3** [yaqwsx/PcbDraw#76] [Q&A] [answered]
I try again to use one of their tools again, and I have problems again. It is not clear to me if I am adding the path correctly.

I downloaded your lib repo that has the basic components but I still don't have the output.

![Con titulo](https://i.imgur.com/ap6iP4q.png )

**Comment by yaqwsx** [comment, marked as answer]
Don't specify individual SVG files as a footprint; you should specify the top level directory; that is `/home/oem/repas/PcbDraw-Lib/KiCAD-base` in your case

**Reply by picli3** [reply]
I see, thank you very much at night I try it, I will write a post and of course I will create new components.
